### PR TITLE
[keyboard] add bear_face/v3, bear_face keyboard folder maintenance

### PR DESCRIPTION
**Pull request by will-hedges**
## Description

* Added `bear_face/v3`
* Updated `info.json`s, keymaps to include `LAYOUT`, `LAYOUT_all` where applicable
* Updated bootloader options, imgur link in readme
* Updated keyboard-level files to be consistent with PR checklist
* Updated copyrights with github username

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bear_face/v3/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            }
+            break;
+
+        case BASE_DVOR:
+            if (record->event.pressed) {
+            // when keycode BASE_DVOR is pressed
+                set_single_persistent_default_layer(_DVOR);
+            } else {
+            // when keycode BASE_DVOR is released
+            }
```
Comment (drashna) [inline keyboards/bear_face/v3/keymaps/default/keymap.c:113]:
```suggestion
            if (record->event.pressed) {
                set_single_persistent_default_layer(_QWER);
            }
            break;
        case BASE_COLE:
            if (record->event.pressed) {
                set_single_persistent_default_layer(_COLE);
            }
            break;
        case BASE_DVOR:
            if (record->event.pressed) {
                set_single_persistent_default_layer(_DVOR);
            }
```

File: keyboards/bear_face/v3/keymaps/default/readme.md
Comment (drashna) [inline keyboards/bear_face/v3/keymaps/default/readme.md:1]: This can be removed.

File: keyboards/bear_face/v3/keymaps/default_iso/keymap.c [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            }
+            break;
+
+        case BASE_DVOR:
+            if (record->event.pressed) {
+            // when keycode BASE_DVOR is pressed
+                set_single_persistent_default_layer(_DVOR);
+            } else {
+            // when keycode BASE_DVOR is released
+            }
```
Comment (drashna) [inline keyboards/bear_face/v3/keymaps/default_iso/keymap.c:113]:
```suggestion
            if (record->event.pressed) {
                set_single_persistent_default_layer(_QWER);
            }
            break;
        case BASE_COLE:
            if (record->event.pressed) {
                set_single_persistent_default_layer(_COLE);
            }
            break;
        case BASE_DVOR:
            if (record->event.pressed) {
                set_single_persistent_default_layer(_DVOR);
            }
```

File: keyboards/bear_face/v3/keymaps/default_iso/readme.md
Comment (drashna) [inline keyboards/bear_face/v3/keymaps/default_iso/readme.md:1]: Should be removed.

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by will-hedges** [comment]
Awaiting for re-review, commenting to keep PR open

**Comment by will-hedges** [comment]
Not sure what happened to this branch but will close and re-open off a fresh `master` branch

**Pull request closed without merging**